### DISCUSSION TITLE
I have thousands of lines of Flynt Deprecated error messages if I enable debug mode

### DISCUSSION THREAD

**Discussion by giorgitchanturidze** [flyntwp/flynt#524] [Q&A] [answered]
![image](https://github.com/flyntwp/flynt/assets/101937929/420059d3-ff31-4c63-838d-44f727bfbb37)
[debug.log](https://github.com/flyntwp/flynt/files/13612189/debug.log)

**Comment by aaronmeder** [comment, marked as answer]
Hey @giorgitchanturidze 

Could it be that you are using PHP 8.2?
I'm getting some of the same errors when I switch to it from PHP 8.1.

If you are using the current 2.0 release of Flynt then it should be possible to use PHP 8.2.
Check that your composer.json has:
```json
"require": {
"flyntwp/acf-field-group-composer": "^1.0.3",
"timber/timber": "^2.0"
},
```
and do a `composer update`. 

This seems to resolve most of the errors for me (while you might possibly still get deprecation notices from other WP plugins).

I haven't tried to work with the older Flynt 1.4 based setup on PHP 8.2, since that is using the Timber v1 branch, but it surely works with PHP 8.1 if you have the latest timber/timber v1.23.

Aaron

**Reply by giorgitchanturidze** [reply]
Thank you for your response! 
I am using Flynt 1.4
![image](https://github.com/flyntwp/flynt/assets/101937929/360369c0-536c-448d-9810-ed5bcd431b3b)
<img width="657" alt="image" src="https://github.com/flyntwp/flynt/assets/101937929/f71d1fe7-85fb-4802-ac88-1e21a93c5724">

**Reply by giorgitchanturidze** [reply]
Changed PHP 8.2 to 8.1, and there are no more and more errors.
@aaronmeder Thank you very much. You guys have all created an amazing theme and you are supportive whenever someone needs help. You are all the best.

**Reply by aaronmeder** [reply]
@giorgitchanturidze Glad that works for you!
Thanks a a lot although all the honor for that belongs to the folks at Bleech 🏆 I am also just enjoying the fantastic package!

If you want you can also try to update timber to v2 using their [Migration guide](https://timber.github.io/docs/v2/upgrade-guides/2.0/). I've never had to do it so far, but heard it should not require too much work.

**Reply by giorgitchanturidze** [reply]
@aaronmeder Thanks!
